src/transformers/models/vit_msn/modeling_vit_msn.py

```diff
@@ -644,6 +646,7 @@ def forward(
         >>> # model predicts one of the 1000 ImageNet classes
         >>> predicted_label = logits.argmax(-1).item()
         >>> print(model.config.id2label[predicted_label])
+        LABEL_183
```

Comment:
Thank you @sayakpaul . Just wondering, if there is a plan to update the config on the Hub for this checkpoint, then update this value here?

Reply:
How the config should be updated? Updated with the ImageNet-1k labels?

Reply:
Sorry @sayakpaul , I should ask this question internally, as it is a checkpoint from fackebook `facebook/vit-msn-small`.
But yes, I think it should be updated with ImageNet-1k labels.

Reply:
Sure. When the classification head is updated with the pretrained params, I think we could revisit it.

Reply:
Or maybe you would like to open a Hub PR for that checkpoint?

Reply:
Yeah that is what my plan is i.e., open a PR to the MSN checkpoints on Hub to update the config.